docs/self-managed/operational-guides/multi-region/dual-region-ops.md

```diff
@@ -0,0 +1,1162 @@
+
+```bash
+kubectl --context $CLUSTER_SURVIVING get deployments $HELM_RELEASE_NAME-operate $HELM_RELEASE_NAME-tasklist -n $CAMUNDA_NAMESPACE_SURVIVING
+# NAME               READY   UP-TO-DATE   AVAILABLE   AGE
+# camunda-operate    0/0     0            0           23m
+# camunda-tasklist   0/0     0            0           23m
+```
+
+For the Zeebe Elasticsearch exporters, there's currently no API available to confirm this. Only the response code of `204` indicates a successful disabling.
+
+</div>
+  </TabItem>
+  <TabItem value="step3" label="Step 3">
+
+#### Create and restore Elasticsearch backup
+
+<StateContainer
+current={<Ten viewBox="140 40 680 500" />}
+desired={<Eleven viewBox="140 40 680 500" />}
+/>
+
+<div>
+
+#### Current state
+
+The Camunda components are currently not reachable by end-users and will not process any new process instances. This allows creating a backup of Elasticsearch without losing any data.
+
+#### Desired state
+
+You are creating a backup of the main Elasticsearch instance in the surviving region and restore it in the recreated region. This Elasticsearch backup contains all the data and may take some time to be finished. The failover Elasticsearch instance only contains a subset of the data from after the region loss and is not sufficient to restore this in the new region.
+
+#### How to get there
+
+This builds on top of the [AWS setup](./../../platform-deployment/helm-kubernetes/platforms/amazon-eks/dual-region.md) and assumes the S3 bucket was automatically created as part of the Terraform execution.
+
+:::info
+
+The procedure works for other cloud providers and bare metal. You have to adjust the AWS S3-specific part depending on your chosen backup source for Elasticsearch. Consult the [Elasticsearch documentation on snapshot and restore](https://www.elastic.co/guide/en/elasticsearch/reference/current/snapshot-restore.html) to learn more about this, and specifically the [different supported types](https://www.elastic.co/guide/en/elasticsearch/reference/current/snapshots-register-repository.html#ess-repo-types) by Elasticsearch.
+
+:::
+
+1. Determine the S3 bucket name by retrieving it via Terraform. Go to `aws/dual-region/terraform` within the repository and retrieve the bucket name from the Terraform state:
+
+```bash
+export S3_BUCKET_NAME=$(terraform output -raw s3_bucket_name)
+```
+
+2. Configure Elasticsearch backup endpoint in the surviving namespace `CAMUNDA_NAMESPACE_SURVIVING`:
+
+```bash
+ELASTIC_POD=$(kubectl --context $CLUSTER_SURVIVING get pod --selector=app\.kubernetes\.io/name=elasticsearch -o jsonpath='{.items[0].metadata.name}' -n $CAMUNDA_NAMESPACE_SURVIVING)
+kubectl --context $CLUSTER_SURVIVING exec -n $CAMUNDA_NAMESPACE_SURVIVING -it $ELASTIC_POD -- curl -XPUT "http://localhost:9200/_snapshot/camunda_backup" -H "Content-Type: application/json" -d'
```

Comment:
shall the container name be used here explicitly?
tried the command, and got this warning:
```
Defaulted container "elasticsearch" out of: elasticsearch, sysctl (init)
```

Reply:
(same for other `kubectl exec commands`)

Reply:
can also keep it as is, not a big deal

Reply:
don't have a strong preference since it will default to the 1st container, which typically is the main one.
Nevertheless, to be explicit added the `-c elasticsearch`.